src/Visualization/Python/GenerateVideo.py

```diff
@@ -0,0 +1,312 @@
+    volulmeFilesxmfDisplay.ScaleTransferFunction.Points = [
+        0.4745234251022339, 0.0, 0.5, 0.0, 0.996696412563324, 1.0, 0.5, 0.0
+    ]
+    # init the 'PiecewiseFunction' selected for 'OpacityTransferFunction'
+    volulmeFilesxmfDisplay.OpacityTransferFunction.Points = [
+        0.4745234251022339, 0.0, 0.5, 0.0, 0.996696412563324, 1.0, 0.5, 0.0
+    ]
+    # reset view to fit data
+    renderView1.ResetCamera(False)
+    # get the material library
+    materialLibrary1 = GetMaterialLibrary()
+    # show color bar/color legend
+    volulmeFilesxmfDisplay.SetScalarBarVisibility(renderView1, True)
+    # update the view to ensure updated data information
+    renderView1.Update()
+    # create a new 'Slice'
+    slice1 = Slice(registrationName='Slice1', Input=volulmeFilesxmf)
+    slice1.SliceType = 'Plane'
+    slice1.HyperTreeGridSlicer = 'Plane'
+    slice1.SliceOffsetValues = [0.0]
+    # Properties modified on slice1.SliceType
+    slice1.SliceType.Normal = [0.0, 0.0, 1.0]
+    # show data in view
+    slice1Display = Show(slice1, renderView1, 'GeometryRepresentation')
+    # init the 'PiecewiseFunction' selected for 'ScaleTransferFunction'
+    slice1Display.ScaleTransferFunction.Points = [
+        0.4745234251022339, 0.0, 0.5, 0.0, 0.996696412563324, 1.0, 0.5, 0.0
+    ]
+    # init the 'PiecewiseFunction' selected for 'OpacityTransferFunction'
+    slice1Display.OpacityTransferFunction.Points = [
+        0.4745234251022339, 0.0, 0.5, 0.0, 0.996696412563324, 1.0, 0.5, 0.0
+    ]
+    # hide data in view
+    Hide(volulmeFilesxmf, renderView1)
+    # show color bar/color legend
+    slice1Display.SetScalarBarVisibility(renderView1, True)
+    # update the view to ensure updated data information
+    renderView1.Update()
+    # Properties modified on lapseLUT
+    lapseLUT.Discretize = 0
+    # Properties modified on slice1
+    slice1.Triangulatetheslice = 0
+    # update the view to ensure updated data information
+    renderView1.Update()
+    # create a new 'Warp By Scalar'
+    warpByScalar1 = WarpByScalar(registrationName='WarpByScalar1',
+                                 Input=slice1)
+    warpByScalar1.Scalars = ['POINTS', 'Lapse']
+    # Properties modified on warpByScalar1
+    warpByScalar1.Scalars = ['POINTS', 'SpatialRicciScalar']
+    warpByScalar1.ScaleFactor = 2.5
+    warpByScalar1.Normal = [0.0, 0.0, -1.0]
+    # show data in view
+    warpByScalar1Display = Show(warpByScalar1, renderView1,
+                                'GeometryRepresentation')
+    # init the 'PiecewiseFunction' selected for 'ScaleTransferFunction'
+    warpByScalar1Display.ScaleTransferFunction.Points = [
+        0.4745234251022339, 0.0, 0.5, 0.0, 0.996696412563324, 1.0, 0.5, 0.0
+    ]
+    # init the 'PiecewiseFunction' selected for 'OpacityTransferFunction'
+    warpByScalar1Display.OpacityTransferFunction.Points = [
+        0.4745234251022339, 0.0, 0.5, 0.0, 0.996696412563324, 1.0, 0.5, 0.0
```

Comment:
You use this transfer function a lot. Store it in a variable and explain it a but.

Reply:
This seems to be a default setting for Paraview that it decided to track in the script. However, I tried taking them out completely and nothing changed, so I think all of these can be safely taken out